cluster/sync-common.sh

```diff
@@ -1,5 +1,5 @@
 function eventually {
-    timeout=$(( $KUBEVIRT_NUM_NODES * 10 ))
+    timeout=$(( $KUBEVIRT_NUM_NODES * 30 ))
```

Comment:
btw there is a little bug i think on line 9 `if [ $timeout -le 0 ]; then`
lets say the timeout is 5, and the interval is 5, it means it will try one time, sleep and then exit
it won't try the 2nd time,
should i change it to `-lt` instead `-le` ?

Reply:
Right, can you a prepare a PR for that ?

Reply:
sure thx

Reply:
https://github.com/nmstate/kubernetes-nmstate/pull/741